src/wazuh_db/schema_global.sql

```diff
@@ -35,7 +35,8 @@ CREATE TABLE IF NOT EXISTS agent (
     `group` TEXT DEFAULT 'default',
     sync_status TEXT NOT NULL CHECK (sync_status IN ('synced', 'syncreq')) DEFAULT 'synced',
     connection_status TEXT NOT NULL CHECK (connection_status IN ('pending', 'never_connected', 'active', 'disconnected')) DEFAULT 'never_connected',
-    disconnection_time INTEGER DEFAULT 0
+    disconnection_time INTEGER DEFAULT 0,
+    group_config_status TEXT NOT NULL CHECK (group_config_status IN ('synced', 'not synced')) DEFAULT 'not synced'
```

Comment:
It looks good, but in line 37 never_conected use underscore as 2 words union, Does not synced should use underscore as union?